load("@io_bazel_rules_go//go:def.bzl", "go_test")

go_test(
    name = "casetest_test",
    timeout = "moderate",
    srcs = [
        "integration_test.go",
        "main_test.go",
        "plan_test.go",
        "stats_test.go",
        "tiflash_selection_late_materialization_test.go",
    ],
    data = glob(["testdata/**"]),
    flaky = True,
<<<<<<< HEAD
    shard_count = 23,
=======
    shard_count = 25,
>>>>>>> d2d12574a35 (planner: fix privilege for the view in the CTE and  wrong error handle (#53556))
    deps = [
        "//pkg/domain",
        "//pkg/errno",
        "//pkg/parser",
        "//pkg/parser/model",
        "//pkg/planner/core",
        "//pkg/planner/property",
        "//pkg/testkit",
        "//pkg/testkit/testdata",
        "//pkg/testkit/testmain",
        "//pkg/testkit/testsetup",
        "//pkg/util/hint",
        "//pkg/util/plancodec",
        "@com_github_pingcap_failpoint//:failpoint",
        "@com_github_stretchr_testify//require",
        "@org_uber_go_goleak//:goleak",
    ],
)
